### ISSUE TITLE
Drag&Drop issues

### ISSUE THREAD

**Issue by VibeDesign** [Jako/SuperBoxSelect#25]
I'm having trouble sorting the items via drag&drop. I encountered the following problems:

1. I can only drag items by clicking and holding the "X" on the item, otherwise it's selecting the actual text of the item
2. After once moving an item via drag&drop, the selection is always locked to that item. Meaning whichever item is selected after moving the first one, will always cause the initial one to be moved and not the selected.
3. In stacked mode, it's possible to (accidentally) drag an item in the empty line below the current selection, causing the empty line to move up which afterwards can't be removed anymore (also not by clearing the whole input). This is also possible in non-stacked mode where an empty space is created (visible by dragging items after it).
4. Seemingly randomly disappearing "X" on list items which are then unable to be removed (also not by clearing the whole input)

MODX 2.8.1-pl
Firefox 85.0

**Comment by VibeDesign** [comment]
Just noticed that it's also not saving any changes after reordering an (one) item to another position.

**Comment by Jako** [comment]
Then there should occur some JS error in the browser console. Which one?

**Comment by VibeDesign** [comment]
I get this error after dragging an item:
```
Uncaught TypeError: j.parentNode is null
    Ne [...]/assets/components/superboxselect/js/mgr/superboxselect.min.js?v=v2.4.1-pl2:7
```

**Comment by Jako** [comment]
Fixed with 2.4.2

**Comment by VibeDesign** [comment]
Thanks a lot for this fast fix!
